frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -377,8 +377,10 @@ local footerTextGeneratorMap = {
     page_turning_inverted = function(footer)
         local symbol_type = footer.settings.item_prefix
         local prefix = symbol_prefix[symbol_type].page_turning_inverted
-        if G_reader_settings:isTrue("input_invert_page_turn_keys") or G_reader_settings:isTrue("input_invert_left_page_turn_keys") or
-           G_reader_settings:isTrue("input_invert_right_page_turn_keys") or G_reader_settings:isTrue("inverse_reading_order") then
+        if G_reader_settings:isTrue("input_invert_page_turn_keys") or
+           G_reader_settings:isTrue("input_invert_left_page_turn_keys") or
+           G_reader_settings:isTrue("input_invert_right_page_turn_keys") or
+           footer.ui.view.inverse_reading_order then
```

Comment:
This feels a bit suspicious. If `G_reader_settings:isTrue("inverse_reading_order")` won't do the trick that may well be a problem because the menu also depends on that for example. In which case it should presumably also be updated here to resolve the problem globally:

https://github.com/koreader/koreader/blob/698fc35c3f1a10233cf69085a87fb912e2f79841/frontend/apps/reader/modules/readerview.lua#L1136-L1142

But that's just a quick note before heading off to bed; I may have overlooked something.

Reply:
Hi @Frenzie , inverse_reading_order is a per-document setting, that is, you can set different values for each book. This makes sense as different books may use different languages and layouts. The global setting of inverse_reading_order is more of a default value.

ReaderView loads and saves the per-document value:

https://github.com/koreader/koreader/blob/698fc35c3f1a10233cf69085a87fb912e2f79841/frontend/apps/reader/modules/readerview.lua#L887

https://github.com/koreader/koreader/blob/698fc35c3f1a10233cf69085a87fb912e2f79841/frontend/apps/reader/modules/readerview.lua#L1047

A more illustrative reference is:

https://github.com/koreader/koreader/blob/698fc35c3f1a10233cf69085a87fb912e2f79841/frontend/ui/elements/page_turns.lua#L117-L126

This code snippet:
- Appends ★ to the menu text when the global default value is true.
- Checks the menu when the per-document value is true, which may or may not equals to the global value.

@Commodore64user FYI.

Reply:
> inverse_reading_order is a per-document setting

I don't think that's a relevant detail wrt my comment, but that it does it the same way elsewhere may be fair enough.